lib/reducers/auth/store_auth_step.dart

```diff
@@ -2,6 +2,7 @@ import 'package:crowdleague/actions/auth/store_auth_step.dart';
 import 'package:crowdleague/models/app/app_state.dart';
 import 'package:redux/redux.dart';
 
+/// Stores [AuthStep] that toggles UI in [AuthPage] and [EmailAuthOptionsPage]
```

Comment:
added this comment to help me understand what effects this reducer has on the app, or how its job is important in the big picture. 
happy to remove if unneccessary :)

Reply:
Great addition, thanks!